### DISCUSSION TITLE
Руководство ПЛИ, п. 3.2 Пациент

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#269] [Лабораторная РГ]
### 3.2 Информация о пациенте recordTarget, patientRole
У1-12: ClinicalDocument ОБЯЗАН иметь один [1..1] элемент recordTarget который
1. ОБЯЗАН иметь один [1..1] элемент patientRole
<img width="307" alt="image" src="https://github.com/user-attachments/assets/3eab52f3-ff2d-402c-96a4-4d42eb7e24d8">

Разбор данного вопроса см [после пункта 3.2.1.з](https://github.com/fhir-ru/core/discussions/269#discussioncomment-10736422)

**Comment by eugenekogan-spb** [comment]
### 3.2.1а  Уникальный идентификатор пациента в МИС
ОБЯЗАН иметь один [1..1] элемент id[1], который соответствует уникальному идентификатору пациента в информационной системе (атрибут @root должен быть сформирован по правилу формирования идентификаторов пациентов "OID_медицинской_организации.100.НомерМИС.НомерЭкзМИС.10", атрибут @extension должен быть представлен не пустым значением);
<img width="744" alt="image" src="https://github.com/user-attachments/assets/b9e3a411-205a-4d18-bd69-6fc082183a17">

**Reply by eugenekogan-spb** [reply]
Предложение В.Родионова [из полного примера ниже](https://github.com/fhir-ru/core/discussions/269#discussioncomment-10845150)

```
//identifier[misPatientId] (OID_медицинской_организации.100.НомерМИС.НомерЭкзМИС.10)
* identifier[0]
  * value = "3421"
  * system = "1.2.643.5.1.13.13.12.2.22.1576.100.1.1.10" 
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#PRN
```

**Reply by Vrodvi** [reply]
Только я там промахнулся - не `#PRN` (это идентификатор поставщика мед.помощи), а `#PN`  

![image](https://github.com/user-attachments/assets/8ed69648-7c85-4b02-8c10-a29635f5d235)

**Comment by eugenekogan-spb** [comment]
### 3.2.1б СНИЛС
ОБЯЗАН иметь один [1..1] элемент id[2], который соответствует страховому номеру индивидуального лицевого счёта (СНИЛС) (атрибут @root должен иметь значение "1.2.643.100.3", атрибут @extension должен быть представлен не пустым значением);
<img width="496" alt="image" src="https://github.com/user-attachments/assets/bae60c62-366f-4b75-8c92-883f31c81919">

**Reply by eugenekogan-spb** [reply]
Предложение В.Родионова [из полного примера ниже](https://github.com/fhir-ru/core/discussions/269#discussioncomment-10845150)

```
//* identifier[SNILS] 
* identifier[+]
  * value = "152-873-614 14"
  * system = "http://fhir.ru/core/systems/snils"
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#SB
```

**Comment by eugenekogan-spb** [comment]
### 3.2.1.в Документ, удостоверяющий личность
в. ДОЛЖЕН иметь один [1..1] элемент identity:IdentityDoc, который:
I. ОБЯЗАН иметь один [1..1] элемент identity:IdentityCardType (атрибут @xsi:type должен иметь значение "CD", атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.48", @codeSystemName должен иметь значение "Документы, удостоверяющие личность", атрибуты @code, @codeSystemName, @displayName, @codeSystemVersion должны быть представлены не пустыми значениями);
II. ДОЛЖЕН иметь один [1..1] элемент identity:Series (атрибут @xsi:type должен иметь значение "ST") с непустым наполнением;
III.ОБЯЗАН иметь один [1..1] элемент identity:Number (атрибут @xsi:type должен иметь значение "ST") с непустым наполнением;
IV.ДОЛЖЕН иметь один [1..1] элемент identity:IssueOrgName (атрибут @xsi:type должен иметь значение "ST") с непустым наполнением;
V. ДОЛЖЕН иметь один [1..1] элемент identity:IssueOrgCode (атрибут @xsi:type должен иметь значение "ST") с непустым наполнением;
VI.ОБЯЗАН иметь один [1..1] элемент identity:IssueDate (атрибут @xsi:type должен иметь значение "TS") с непустым наполнением, атрибут @value должен быть представлен не пустым значением и содержать дату выдачи документа);
<img width="734" alt="image" src="https://github.com/user-attachments/assets/b7076588-bccc-45ea-b144-113334547dbc">

**Reply by eugenekogan-spb** [reply]
Предложение В.Родионова [из полного примера ниже](https://github.com/fhir-ru/core/discussions/269#discussioncomment-10845150)
```
//* identifier[passportRF]
* identifier[+]
  * value = "7412 809982"
  * system = "http://fhir.ru/core/systems/passport-RF"
  * type = Core_Cs_Nsi_Identity_Document#1
```

**Reply by eugenekogan-spb** [reply]
Решение о пробеле между серией и номером согласовано на встрече 17.01.25 со следующим комментарием:
- бывают документы, где внутри серии есть пробел, например в паспорте или водительском удостоверении две цифры-пробел-две цифры, однако этот пробел фактически опускают, в пример СЭМД он также опущен
- в руководство необходимо внести фразу "при наличии пробела внутри серии этот пробел сокращаем".

**Reply by eugenekogan-spb** [reply]
Необходимо добавить дату выдачи и выдавшую организацию
```
//* identifier[passportRF]
* identifier[+]
  * value = "7412 809982"
  * system = "http://fhir.ru/core/systems/passport-RF"
  * type = Core_Cs_Nsi_Identity_Document#1
  * period
    * start = "2005-02-08"
  * assigner
    * display = "Отделом внутренних дел Ленинского района гор. Барнаула"
    * identifier
      * value = "222-070"
```

**Reply by eugenekogan-spb** [reply]
Код подразделения является, по сути, идентификатором организации, выдавшей документ.
Это решение согласовано на встрече 17.01.25.

**Comment by eugenekogan-spb** [comment]
### 3.2.1г Полис ОМС

г. ДОЛЖЕН иметь один [1..1] элемент identity:InsurancePolicy, который:
I. ОБЯЗАН иметь один [1..1] элемент identity:InsurancePolicyType (атрибут @xsi:type должен иметь значение "CD", атрибут @codeSystem должен быть представлен и иметь значение "1.2.643.5.1.13.13.11.1035", атрибут @codeSystemName должен иметь значение "Виды полиса обязательного медицинского страхования", атрибуты @code, @displayName, @codeSystemVersion должны быть представлены не пустыми значениями);
II. МОЖЕТ иметь один [0..1] элемент identity:Series (атрибут @xsi:type должен иметь значение "ST") с непустым наполнением;
III.ОБЯЗАН иметь один [1..1] элемент identity:Number (атрибут @xsi:type должен иметь значение "ST") с непустым наполнением;

<img width="750" alt="image" src="https://github.com/user-attachments/assets/f694a482-ddef-4fce-b01a-14a98d41b82d">

**Reply by eugenekogan-spb** [reply]
Предложение В.Родионова [из полного примера ниже](https://github.com/fhir-ru/core/discussions/269#discussioncomment-10845150)
```
* identifier[+]
  * value = "123456 1234567890"
  * system = "http://fhir.ru/core/systems/OMS"
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#SN
```

**Reply by eugenekogan-spb** [reply]
Видимо, целесообразно сохранить типизацию по FHIR, показав, что это полис. Но дополнительно надо закодировать тип полиса по НСИ МЗ РФ.
```
* identifier[+]
  * value = "9876543211234567"
  * system = "http://fhir.ru/core/systems/OMS"
  * type
    * text = "Полис ОМС единого образца, бессрочный"
    * coding[+]
      * system = "http://terminology.hl7.org/CodeSystem/v2-0203"
      * code = #SN
    * coding[+]
      * system = "1.2.643.5.1.13.13.11.1035"
      * code = #2
```

**Comment by eugenekogan-spb** [comment]
### 3.2.1.д Адрес

д. ДОЛЖЕН иметь хотя бы один элемент [1..2] addr, который:
I. ОБЯЗАН иметь элемент один [1..1] address:Type (атрибут @xsi:type должен иметь значение "CD", атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.11.1504", атрибуты @code, @codeSystemVersion, @codeSystemName, @displayName должны иметь непустое наполнение);
II. обязан соответствовать требованиям к элементу addr, описанными в разделе "Общие ограничения";

<img width="764" alt="image" src="https://github.com/user-attachments/assets/ab635adb-748e-4743-b309-57671bdb3111">
<img width="761" alt="image" src="https://github.com/user-attachments/assets/8f26f592-e70e-4dc2-9368-02b8be64fbc3">

**Reply by Vrodvi** [reply]
В корневом профиле адреса не обнаружил способа обозначить тип адреса по справочнику https://nsi.rosminzdrav.ru/dictionaries/1.2.643.5.1.13.13.11.1504/passport/1.4

**Reply by Vrodvi** [reply]
связанное общее ограничение - https://github.com/fhir-ru/core/discussions/267#discussioncomment-10328388

**Reply by eugenekogan-spb** [reply]
Результрующее предложение В.Родионова [из полного примера ниже](https://github.com/fhir-ru/core/discussions/269#discussioncomment-10845150)

```
* address 

  * text = "Россия, г. Санкт-Петербург, Софийская ул., д.32-1-987"
  
  // * extension[addressType]
  * extension[0]
    * url = "addressType"
    * valueCoding
      * code = #1
      * system = "1.2.643.5.1.13.13.11.1504"
      * version = "1.4"         
      * display = "Адрес по месту жительства (постоянной регистрации)"

  //* extension[regionRF]
  * extension[+]
    * url = "regionRF"
    * valueCoding
      * code = #77
      * system = "1.2.643.5.1.13.13.99.2.20"
      * version = "6.3"         
      * display = "г. Москва"

  * postalCode = "192236"

  //* extension[fias]
  * extension[+]
    * url = "fias"
    //* extension[aoguid]
    * extension[0]
      * url = "aoguid"
      * valueIdentifier
        * value = "7b1996be-c0fe-4797-bedc-f503e31a83fd"
        * system = "urn:hl7-ru:fias:aoguid"
    //* extension[houseguid]
    * extension[+]
      * url = "houseguid"
      * valueIdentifier
        * value = "7b1996be-c0fe-4797-bedc-f503e31a83fd"
        * system = "urn:hl7-ru:fias:houseguid"
```

**Comment by eugenekogan-spb** [comment]
### 3.2.1.е контакты

е. МОЖЕТ иметь произвольное количество [0..*] элементов telecom. Хотя бы один telecom СЛЕДУЕТ указать со схемой "tel:";

<img width="514" alt="image" src="https://github.com/user-attachments/assets/ad235534-71a0-4eee-82eb-d09937dd9d9e">

**Reply by eugenekogan-spb** [reply]
Предложение В.Родионова [из полного примера ниже](https://github.com/fhir-ru/core/discussions/269#discussioncomment-10845150)
```
* telecom[0]
  * system = #phone
  * value = "+7 960 700 5745"
  * use = #mobile
```

**Comment by eugenekogan-spb** [comment]
### 3.2.1.ж Личные данные

ж. ОБЯЗАН иметь один [1..1] элемент patient, который:
I. ОБЯЗАН содержать один [1..1] элемент name, заполненный согласно общему
ограничению "Имена и наименования <name>";
II. ДОЛЖЕН содержать один [1..1] элемент administrativeGenderCode (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.11.1040", атрибут @codeSystemName должен иметь значение "Пол пациента", атрибуты @code, @displayName, @codeSystemVersion должны быть представлены не пустыми значениями);
III.ДОЛЖЕН содержать один [1..1] элемент birthTime (атрибут @value должен быть заполнен согласно общему ограничению "Дата и время" и содержать в себе дату рождения с точностью до дня);

<img width="669" alt="image" src="https://github.com/user-attachments/assets/57449ba1-f908-4577-b9f2-9b7ad6cc3e35">

**Reply by eugenekogan-spb** [reply]
Предложение В.Родионова [из полного примера ниже](https://github.com/fhir-ru/core/discussions/269#discussioncomment-10845150)

```
* name
  * family = "Габсбург-Лотарингская"
  * given[0] = "Мария-Антуанетта"
  * given[1] = "Иоганновна"
  * use = #official

* gender = #female

* birthDate = "1955-01-10"
```

**Reply by eugenekogan-spb** [reply]
Необходимо оговорить, что для кодирования пола требуется маппинг.
Он позволит перекодировщику CDA-FHIR указать идентификатор справочника НСИ и код по справочнику.

**Comment by eugenekogan-spb** [comment]
### 3.2.1.з Медицинская организация, оформившая протокол лабораторного исследования

з. ОБЯЗАН иметь [1..1] один элемент providerOrganization, который:
I. ОБЯЗАН иметь один [1..1] элемент id[1] (атрибут @root должен быть представлен с не пустым значением (по справочнику "Реестр медицинских организаций Российской Федерации" (OID: 1.2.643.5.1.13.13.11.1461), поле "OID"), элемент может иметь атрибут @extension, который должен быть представлен с непустым значением, если имеется информация о структурном подразделении медицинской организации (по справочнику "ФРМО. Справочник структурных подразделений" (OID: 1.2.643.5.1.13.13.99.2.114), поле "OID структурного подразделения")), содержащий уникальный идентификатор медицинской организации и уникальный идентификатор структурного подразделения;
II. МОЖЕТ содержать один [0..1] элемент id[2], который соответствует лицензии на осуществление медицинской деятельности (атрибут @root должен иметь значение "1.2.643.5.1.13.2.1.1.1504.101", атрибут @extension должен содержать номер лицензии, @assigningAuthorityName должен содержать наименование организации, выдавшей лицензию, и дату выдачи лицензии);
III.ДОЛЖЕН иметь один [1..1] элемент identity:Props. (в случае, если реквизиты отсутствуют, указывается атрибут @nullFlavor), который:
1. МОЖЕТ содержать один [0..1] элемент identity:Ogrn (атрибут @xsi:type должен иметь значение "ST") с непустым наполнением.Текущее поле не может быть заполнено совместно с identity:Ogrnip;
2. МОЖЕТ содержать один [0..1] элемент identity:Ogrnip (атрибут @xsi:type должен иметь значение "ST") с непустым наполнением.Текущее поле не может быть заполнено совместно с identity:Ogrn;
3. МОЖЕТ содержать один [0..1] элемент identity:Okpo (атрибут @xsi:type
должен иметь значение "ST") с непустым наполнением;
4. МОЖЕТ содержать один [0..1] элемент identity:Okato (атрибут @xsi:type
должен иметь значение "ST") с непустым наполнением;

IV.ОБЯЗАН содержать один [1..1] элемент name, содержащий наименование
организации;
V. МОЖЕТ иметь произвольное количество [0..*] элементов telecom, содержащих контакты организации, хотя бы один telecom СЛЕДУЕТ указать со схемой "tel:";
VI.ОБЯЗАН содержать один [1..1] элемент addr, заполненный согласно общему ограничению "Адреса <addr>".

<img width="762" alt="image" src="https://github.com/user-attachments/assets/bb641299-7d44-4e26-aae6-22a837741a5a">

**Reply by Vrodvi** [reply]
В СЭМД/CDA несовсем корректно считать `providerOrganization` как `Медицинская организация, оформившая протокол лабораторного исследования`, т.к. в CDA providerOrganization - это организация, в которой ведётся карта указанного пациента. Да, с большой вероятностью документ будет создан в этой же организации, но это не обязательно.

В качестве аналога `recordTarget.providerOrganization` в FHIR используется Patient.managingOrganization.

А для `Медицинская организация, оформившая протокол лабораторного исследования` больше подходит участник `custodian` из CDA и его аналог `Composition.custodian` в FHIR.

Также `Медицинская организация, оформившая протокол лабораторного исследования` можно считать ораганизацию автора документа. В СЭМД/CDA информация об одной и той же организации продублирована во все трёх местах - recordTarget.providerOrganization, custodian, assignedAuthor.representedOrganization. В FHIR можно один раз определить организацию и сослаться на неё из всех трёх мест - из ресурса пациента, автора, хранителя документа (custodian).

Пример:
```
Instance: document-1
InstanceOf: Bundle
Usage: #example

* type = #document

* entry[0]
  * fullUrl = "[base]/Composition/composition-id-2"
  * resource = composition-id-2

* entry[+]
  * fullUrl = "[base]/Patient/patient-id-3"
  * resource = patient-id-3

* entry[+]
  * fullUrl = "[base]/Organization/organization-id-4"
  * resource = organization-id-4

* entry[+]
  * fullUrl = "[base]/PractitionerRole/practitionerrole-id-5"
  * resource = practitionerrole-id-5

Instance: composition-id-2
InstanceOf: Composition
Usage: #inline

* status = #final
* type = http://loinc.org#11502-2
* date = "2013-02-01T12:30:02Z"
* title = "Результат лабораторного исследования"
* author = Reference(practitionerrole-id-5)

* subject = Reference(patient-id-3) "Иванов Иван"

* custodian = Reference(organization-id-4)

Instance: patient-id-3
InstanceOf: Patient
Usage: #inline

* active = true

* name
  * text = "Иванов Иван"
  
* managingOrganization = Reference(organization-id-4)
  
Instance: organization-id-4
InstanceOf: Organization
Usage: #inline

* name = "ГБУЗ Алейская ЦРБ"

Instance: practitionerrole-id-5
InstanceOf: PractitionerRole
Usage: #inline

* organization = Reference(organization-id-4)

```

**Reply by eugenekogan-spb** [reply]
Для атрибутов организации: предложение В.Родионова [из полного примера ниже](https://github.com/fhir-ru/core/discussions/269#discussioncomment-10845150)

```
* name = "ОБЩЕСТВО С ОГРАНИЧЕННОЙ ОТВЕТСТВЕННОСТЬЮ \"ХЕЛИКС АЛТАЙ\""

//<!-- [0..*] Контакты медицинской организации или ИП -->
* telecom[0]
  * system = #phone
  * value = "+7 960 700 5745"
  * use = #mobile

//<!-- R [1..1] Адрес медицинской организации или ИП -->
* address
  //<!-- R [1..1] Адрес (текстом) -->
  * text = "190005, Россия, г. Санкт-Петербург, наб. р. Фонтанки, д. 132"

  //<!-- R [1..1] Код субъекта РФ -->
  //* extension[regionRF]
  * extension[+]
    * url = "regionRF"
    * valueCoding
      * code = #2
      * system = "1.2.643.5.1.13.13.99.2.20"
      * version = "6.3"         
      * display = "Алтайский край"
        
  //<!-- [1..1] Почтовый индекс -->
  * postalCode = "656067"
  
  
  ////<!-- [1..1] Идентификатор адреса по ФИАС -->
  //* extension[fias]
  * extension[+]
    * url = "fias"
    ////<!-- R [1..1] Глобальный уникальный идентификатор адресного объекта -->
    //* extension[aoguid]
    * extension[0]
      * url = "aoguid"
      * valueIdentifier
        * value = "796de636-5fd9-4f3d-97ad-c865c60a024d"
        * system = "urn:hl7-ru:fias:aoguid"
    ////<!-- [1..1] Глобальный уникальный идентификатор дома -->
    //* extension[houseguid]
    * extension[+]
      * url = "houseguid"
      * valueIdentifier
        * value = "19229ca9-d457-4f07-ab86-8d64c868502a"
        * system = "urn:hl7-ru:fias:houseguid"

//<!-- [0..1] Код ОГРНИП -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/ОГРНИП"
  * value = "4719008550"
  
//<!-- [0..1] Код ОКПО -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/ОКПО"
  * value = "03703245"
  
//<!-- [0..1] Код ОКАТО -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/ОКАТО"
  * value = "01401364000"  

//<!-- R [1..1] Уникальный идентификатор медицинской организации и (при наличии) уникальный идентификатор структурного подразделения -->
//* identifier[FRMO]
* identifier[+]
  * system = "http://fhir.ru/core/systems/frmo"
  * value = "1.2.643.5.1.13.13.12.2.22.27143"

//<!-- [0..1] Сведения о лицензии на осуществление медицинской деятельности -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/medlicense"
  * value = "Л041-01151-22/00376231"
  * assigner
    * display = "Министерство здравоохранения Алтайского края. Дата регистрации: 29.05.2020"

```

**Comment by Vrodvi** [comment]
> recordTarget который
> 
> 1. ОБЯЗАН иметь один [1..1] элемент patientRole

Конструкция recordTarget.patientRole тоже требует маппинга на FHIR.
В данном случае указание, что ресурс Patient является целью для описываемой в документе информации, должно быть выражено через атрибут subject ресурса Composition.

К примеру:
```
Instance: document-1
InstanceOf: Bundle
Usage: #example

* type = #document

* entry[0]
  * fullUrl = "[base]/Composition/composition-id-1"
  * resource = composition-id-1

* entry[+]
  * fullUrl = "[base]/Patient/patient-id-1"
  * resource = patient-id-1


Instance: composition-id-1
InstanceOf: Composition
Usage: #inline

* subject = Reference([base]/Patient/patient-id-1) "Иванов Иван"


Instance: patient-id-1
InstanceOf: Patient
Usage: #inline

* active = true

* name
  * text = "Иванов Иван"
```

**Comment by Vrodvi** [comment]
Полный пример по блоку "3.2 Пациент"
```
Instance: document-1
InstanceOf: Bundle
Usage: #example

* type = #document

* entry[0]
  * fullUrl = "[base]/Composition/composition-id-2"
  * resource = composition-id-2

* entry[+]
  * fullUrl = "[base]/Patient/patient-id-3"
  * resource = patient-id-3

* entry[+]
  * fullUrl = "[base]/Organization/organization-id-4"
  * resource = organization-id-4

* entry[+]
  * fullUrl = "[base]/PractitionerRole/practitionerrole-id-5"
  * resource = practitionerrole-id-5

Instance: composition-id-2
InstanceOf: Composition
Usage: #inline

* status = #final
* type = http://loinc.org#11502-2
* date = "2013-02-01T12:30:02Z"
* title = "Результат лабораторного исследования"
* author = Reference(practitionerrole-id-5)

// Информация о пациенте recordTarget
* subject = Reference(patient-id-3) "Иванов Иван"

* custodian = Reference(organization-id-4)

Instance: patient-id-3
InstanceOf: Patient
Usage: #inline

* active = true

* name
  * text = "Иванов Иван"
  
* managingOrganization = Reference(organization-id-4)
  
//3.2.1а Уникальный идентификатор пациента в МИС
//<!-- R [1..1] Уникальный идентификатор пациента в МИС -->
//<id root="1.2.643.5.1.13.13.12.2.22.1576.100.1.1.10" extension="3421"/>
//identifier[misPatientId] (OID_медицинской_организации.100.НомерМИС.НомерЭкзМИС.10)
* identifier[0]
  * value = "3421"
  * system = "1.2.643.5.1.13.13.12.2.22.1576.100.1.1.10" 
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#PRN

//3.2.1б СНИЛС
//* identifier[SNILS] 
* identifier[+]
  * value = "152-873-614 14"
  * system = "http://fhir.ru/core/systems/snils"
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#SB

//3.2.1.в Документ, удостоверяющий личность  
//* identifier[passportRF]
* identifier[+]
  * value = "7412 809982"
  * system = "http://fhir.ru/core/systems/passport-RF"
  * type = Core_Cs_Nsi_Identity_Document#1  
  
//3.2.1г Полис ОМС  
* identifier[+]
  * value = "123456 1234567890"
  * system = "http://fhir.ru/core/systems/OMS"
  * type = http://terminology.hl7.org/CodeSystem/v2-0203#SN
  
//3.2.1.д Адрес
* address 

  * text = "Россия, г. Санкт-Петербург, Софийская ул., д.32-1-987"
  
  // * extension[addressType]
  * extension[0]
    * url = "addressType"
    * valueCoding
      * code = #1
      * system = "1.2.643.5.1.13.13.11.1504"
      * version = "1.4"         
      * display = "Адрес по месту жительства (постоянной регистрации)"

  //* extension[regionRF]
  * extension[+]
    * url = "regionRF"
    * valueCoding
      * code = #77
      * system = "1.2.643.5.1.13.13.99.2.20"
      * version = "6.3"         
      * display = "г. Москва"

  * postalCode = "192236"

  //* extension[fias]
  * extension[+]
    * url = "fias"
    //* extension[aoguid]
    * extension[0]
      * url = "aoguid"
      * valueIdentifier
        * value = "7b1996be-c0fe-4797-bedc-f503e31a83fd"
        * system = "urn:hl7-ru:fias:aoguid"
    //* extension[houseguid]
    * extension[+]
      * url = "houseguid"
      * valueIdentifier
        * value = "7b1996be-c0fe-4797-bedc-f503e31a83fd"
        * system = "urn:hl7-ru:fias:houseguid"

//3.2.1.е контакты
* telecom[0]
  * system = #phone
  * value = "+7 960 700 5745"
  * use = #mobile
  
//3.2.1.ж Личные данные

* name
  * family = "Габсбург-Лотарингская"
  * given[0] = "Мария-Антуанетта"
  * given[1] = "Иоганновна"
  * use = #official

* gender = #female

* birthDate = "1955-01-10"
  
Instance: organization-id-4
InstanceOf: Organization
Usage: #inline

//<!-- R [1..1] Наименование медицинской организации или ФИО ИП -->
* name = "ОБЩЕСТВО С ОГРАНИЧЕННОЙ ОТВЕТСТВЕННОСТЬЮ \"ХЕЛИКС АЛТАЙ\""

//<!-- [0..*] Контакты медицинской организации или ИП -->
* telecom[0]
  * system = #phone
  * value = "+7 960 700 5745"
  * use = #mobile

//<!-- R [1..1] Адрес медицинской организации или ИП -->
* address
  //<!-- R [1..1] Адрес (текстом) -->
  * text = "190005, Россия, г. Санкт-Петербург, наб. р. Фонтанки, д. 132"

  //<!-- R [1..1] Код субъекта РФ -->
  //* extension[regionRF]
  * extension[+]
    * url = "regionRF"
    * valueCoding
      * code = #2
      * system = "1.2.643.5.1.13.13.99.2.20"
      * version = "6.3"         
      * display = "Алтайский край"
        
  //<!-- [1..1] Почтовый индекс -->
  * postalCode = "656067"
  
  
  ////<!-- [1..1] Идентификатор адреса по ФИАС -->
  //* extension[fias]
  * extension[+]
    * url = "fias"
    ////<!-- R [1..1] Глобальный уникальный идентификатор адресного объекта -->
    //* extension[aoguid]
    * extension[0]
      * url = "aoguid"
      * valueIdentifier
        * value = "796de636-5fd9-4f3d-97ad-c865c60a024d"
        * system = "urn:hl7-ru:fias:aoguid"
    ////<!-- [1..1] Глобальный уникальный идентификатор дома -->
    //* extension[houseguid]
    * extension[+]
      * url = "houseguid"
      * valueIdentifier
        * value = "19229ca9-d457-4f07-ab86-8d64c868502a"
        * system = "urn:hl7-ru:fias:houseguid"

//<!-- [0..1] Код ОГРНИП -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/ОГРНИП"
  * value = "4719008550"
  
//<!-- [0..1] Код ОКПО -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/ОКПО"
  * value = "03703245"
  
//<!-- [0..1] Код ОКАТО -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/ОКАТО"
  * value = "01401364000"  

//<!-- R [1..1] Уникальный идентификатор медицинской организации и (при наличии) уникальный идентификатор структурного подразделения -->
//* identifier[FRMO]
* identifier[+]
  * system = "http://fhir.ru/core/systems/frmo"
  * value = "1.2.643.5.1.13.13.12.2.22.27143"

//<!-- [0..1] Сведения о лицензии на осуществление медицинской деятельности -->
* identifier[+]
  * system = "http://fhir.ru/core/systems/medlicense"
  * value = "Л041-01151-22/00376231"
  * assigner
    * display = "Министерство здравоохранения Алтайского края. Дата регистрации: 29.05.2020"


Instance: practitionerrole-id-5
InstanceOf: PractitionerRole
Usage: #inline

* organization = Reference(organization-id-4)

CodeSystem: Core_Cs_Nsi_Identity_Document
Id:         core-cs-nsi-identity-document
* #1 "Паспорт гражданина Российской Федерации"
```
